packages/integration-tests/fixtures/angular-cli-workspace/.eslintrc.js

```diff
@@ -193,6 +193,9 @@ module.exports = {
 
         // ORIGINAL tslint.json -> "template-no-negated-async": true,
         '@angular-eslint/template/no-negated-async': 'error',
+
+        // ORIGINAL tslint.json -> "template-cyclomatic-complexity": false,
```

Comment:
Was this the case? I thought I had captured all the rules that were included in the original tslint.json...

Reply:
You are right, there is just no mention of the rule. I will remove it.